apps/vs-code-designer/src/constants.ts

```diff
@@ -67,6 +67,7 @@ export enum extensionCommand {
   openFile = 'azureLogicAppsStandard.openFile',
   createNewProject = 'azureLogicAppsStandard.createNewProject',
   createNewCodeProject = 'azureLogicAppsStandard.createNewCodeProject',
+  createNewDataMap = 'azureLogicAppsStandard.createNewDataMap',
```

Comment:
NIT: I think you missed the `.dataMap.` in between the command. Or is there a specific logic for when to have it or not?

Reply:
I was thinking createNewDataMap/loadDataMapFile were self-explanatory to be datamap commands so did not add them. But adding them as suggested!